src/script.js

```diff
@@ -880,3 +882,14 @@ scrollIconElement.addEventListener("click", function(e) {
 
 document.getElementById("check").addEventListener("click", check);
 document.getElementById("speak").addEventListener("click", changeSpeak);
+
+ textarea.addEventListener("input", function(e){
+    if(e.target.value) {
```

Comment:
can we use the ternary operator for the conditions? It will look more elegant as well as result in a cleaner code?
for example
```js
micDiv.style.display = e.target.value?"none":"block"
```
maybe?
looks good to me anyways

Reply:
@gabru-md the statements inside if else are multi line(2 lines each), ternary is used for single line statements. Isn't it better this way? Thanks for your review though!

Reply:
@gabru-md Can you update on this comment